cmd/env/command.go

```diff
@@ -40,8 +40,7 @@ their names to upper case and replacing punctuation with underscores. For exampl
       --cassandra.connections-per-host    CASSANDRA_CONNECTIONS_PER_HOST
       --metrics-backend                   METRICS_BACKEND
 
-The following configuration options are only available via environment variables:
-`)
+The following configuration options are only available via environment variables:\n`)
```

Comment:
are you sure `\n` is actually interpreted when in back-tick string?

Reply:
Good question, I don't expect it to be, but let me verify/ replace with something that preserves formatting.

Reply:
just run the cmd and see the output: `go run ./cmd/query/main.go env`

Reply:
\n will be interpreted as the literal backslash and character n since it is raw string literal. 
https://golang.org/ref/spec#String_literals
I would suggest inserting fmt.Fprint(cmd.OutOrStdout(), "\n") after Fprintln, if it has to be changed.

Reply:
Why the complication? My original suggestion was to concatenate with +"\n"